.php_cs

```diff
@@ -0,0 +1,95 @@
+<?php
```

Comment:
If this `.php_cs` is intentionally added, can you add it in a separate PR please?